package.json

```diff
@@ -3,67 +3,67 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@babel/core": "^7.16.0",
-    "@ckeditor/ckeditor5-basic-styles": "^35.4.0",
-    "@ckeditor/ckeditor5-dev-utils": "^32.0.1",
-    "@ckeditor/ckeditor5-editor-classic": "^35.4.0",
-    "@ckeditor/ckeditor5-essentials": "^35.4.0",
-    "@ckeditor/ckeditor5-paragraph": "^35.4.0",
-    "@ckeditor/ckeditor5-react": "^5.0.3",
-    "@ckeditor/ckeditor5-theme-lark": "^35.4.0",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.3",
-    "@svgr/webpack": "^5.5.0",
+    "@babel/core": "^7.22.1",
+    "@ckeditor/ckeditor5-basic-styles": "^38.0.1",
+    "@ckeditor/ckeditor5-dev-utils": "^38.0.0",
+    "@ckeditor/ckeditor5-editor-classic": "^38.0.1",
+    "@ckeditor/ckeditor5-essentials": "^38.0.1",
+    "@ckeditor/ckeditor5-paragraph": "^38.0.1",
+    "@ckeditor/ckeditor5-react": "^6.0.0",
+    "@ckeditor/ckeditor5-theme-lark": "^38.0.1",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
+    "@svgr/webpack": "^8.0.1",
     "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "babel-jest": "^27.4.2",
-    "babel-loader": "^8.2.3",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
+    "babel-jest": "^29.5.0",
+    "babel-loader": "^9.1.2",
     "babel-plugin-named-asset-import": "^0.3.8",
     "babel-preset-react-app": "^10.0.1",
     "bfj": "^7.0.2",
-    "browserslist": "^4.18.1",
-    "camelcase": "^6.2.1",
+    "browserslist": "^4.21.7",
+    "camelcase": "^7.0.1",
     "case-sensitive-paths-webpack-plugin": "^2.4.0",
-    "css-loader": "^6.5.1",
-    "css-minimizer-webpack-plugin": "^3.2.0",
-    "dotenv": "^10.0.0",
-    "dotenv-expand": "^5.1.0",
-    "eslint": "^8.3.0",
+    "css-loader": "^6.8.1",
+    "css-minimizer-webpack-plugin": "^5.0.0",
+    "dotenv": "^16.1.3",
+    "dotenv-expand": "^10.0.0",
+    "eslint": "^8.41.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-webpack-plugin": "^3.1.1",
+    "eslint-webpack-plugin": "^4.0.1",
     "file-loader": "^6.2.0",
-    "fs-extra": "^10.0.0",
-    "html-webpack-plugin": "^5.5.0",
+    "fs-extra": "^11.1.1",
+    "html-webpack-plugin": "^5.5.1",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.4.3",
-    "jest-resolve": "^27.4.2",
-    "jest-watch-typeahead": "^1.0.0",
-    "mini-css-extract-plugin": "^2.4.5",
-    "postcss": "^8.4.4",
+    "jest": "^29.5.0",
+    "jest-resolve": "^29.5.0",
+    "jest-watch-typeahead": "^2.2.2",
+    "mini-css-extract-plugin": "^2.7.6",
+    "postcss": "^8.4.24",
     "postcss-flexbugs-fixes": "^5.0.2",
-    "postcss-loader": "^6.2.1",
+    "postcss-loader": "^7.3.2",
     "postcss-normalize": "^10.0.1",
-    "postcss-preset-env": "^7.0.1",
+    "postcss-preset-env": "^8.4.2",
```

Comment:
I feel that we should bump only `@ckeditor` packages to avoid breaking the environment due to potential breaking changes in other dependencies.

Reply:
Reverted the changes and updated only our deps.